Samples/Windowing/cpp-winui/SampleApp/MainWindow.xaml.h

```diff
@@ -16,12 +16,13 @@ struct MainWindow : MainWindowT<MainWindow>
     void NavigationView_Loaded(winrt::IInspectable const& sender, winrt::RoutedEventArgs const& args);
     void NavigationView_ItemInvoked(winrt::IInspectable const& sender, winrt::NavigationViewItemInvokedEventArgs const& args);
     void NavigationView_BackRequested(winrt::NavigationView const& sender, winrt::NavigationViewBackRequestedEventArgs const& args);
+    void MyWindowIcon_DoubleTapped(winrt::Windows::Foundation::IInspectable const& sender, winrt::Microsoft::UI::Xaml::Input::DoubleTappedRoutedEventArgs const& e);
 
 private:
     winrt::AppWindow GetAppWindowForCurrentWindow();
 
     winrt::AppWindow m_mainAppWindow{ nullptr };
-    hstring m_windowTitle = L"WinUI Desktop C++ Sample App";
+    hstring m_windowTitle = L"WinUI Desktop C++ Sample App";  
```

Comment:
Please remove the trailing space.

Reply:
[eve/windowingfix (#59)](https://github.com/microsoft/WindowsAppSDK-Samples/pull/59/commits/132cdc25bb9030b6c3d2c0add07993e3b6ff726c)
Done.